### PR TITLE
Update intention precedence table in the docs

### PR DESCRIPTION
**Pull request by mkeeler**

### CODE REVIEW

File: website/source/docs/connect/intentions.html.md [hunk 1]
```diff
@@ -111,9 +116,9 @@ valid non-wildcard intentions to match.
 
 The numbers in the table above are not stable. Their ordering will remain
 fixed but the actual number values may change in the future.
-The numbers are non-contiguous because there are
-some unused values in the middle in preparation for a future version of
-Consul supporting namespaces.
+
+-> **Consul Enterprise** - Namespaces are an Enterprise feature. In Consul OSS all of the rows in
```
Comment (kaitlincarter-hc) [inline website/source/docs/connect/intentions.html.md]:
```suggestion
-> **Consul Enterprise** - Namespaces are an Enterprise feature. In Consul OSS any of the rows in
```

File: website/source/docs/connect/intentions.html.md [hunk 2]
```diff
@@ -111,9 +116,9 @@ valid non-wildcard intentions to match.
 
 The numbers in the table above are not stable. Their ordering will remain
 fixed but the actual number values may change in the future.
-The numbers are non-contiguous because there are
-some unused values in the middle in preparation for a future version of
-Consul supporting namespaces.
+
+-> **Consul Enterprise** - Namespaces are an Enterprise feature. In Consul OSS any of the rows in
+the table with a `*` for either the source or destination namespace are not possible.
```
Comment (kaitlincarter-hc) [inline website/source/docs/connect/intentions.html.md]:
```suggestion
the table with a `*` for either the source namespace or destination namespace are not applicable.
```
Does changing it to "applicable" change the meaning?

Comment (mkeeler) [reply website/source/docs/connect/intentions.html.md]: I don’t think it does.

Comment (kaitlincarter-hc) [reply website/source/docs/connect/intentions.html.md]: How would you feel about listing the precedences'? Are they likely to change?

**Pull request merged**